### PR TITLE
Kā anchor tóng-àn kái chò ū teh iōng--ê.

### PR DESCRIPTION
**Pull request by ngoohebi**

### CODE REVIEW

**Comment by ngoohebi** [comment]
@sih4sing5hong5  Kái án-ne khòaⁿ ē-sái bē.

**Comment by sih4sing5hong5** [comment]
我改做直接merge 去master，較明

File: ThoanTaigi.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -264,14 +259,6 @@
...
 			sourceTree = "<group>";
 		};
-		7B5488A71D2DAC9D0056A1BE /* plum */ = {
-			isa = PBXGroup;
-			children = (
-				441E636522B7E90C006DCCDD /* terra_pinyin.schema.yaml */,
-			);
-			name = plum;
-			sourceTree = "<group>";
-		};
```
Comment (sih4sing5hong5) [inline ThoanTaigi.xcodeproj/project.pbxproj:274]: 雖罔 https://github.com/i3thuan5/ThoanTaigi/pull/27/files#diff-5af8d6b6be54cecccb507abdaa3621faR247 有 squirrel.yaml，
敢ē-tàng解說一下這kue̍h `PBXGroup` leh tshòng sannh？
有ê是`name=plum`，有ê是`name = SharedSupport;`

File: package/add_data_files [hunk 2]
```diff
@@ -5,36 +5,42 @@ set -e
...
+# id:  441E63{80}22B7E96F006DCCDD /* terra_pinyin.schema.yaml 
+# ref: 441E63{65}22B7E90C006DCCDD /* terra_pinyin.schema.yaml
+# anchor=(80 65 terra_pinyin.schema.yaml)
+# lastid=80
 
-lastid=80
+# id:  442B5B{88}1570C37200370DEA /* squirrel.yaml
+# ref: 442B5B{87}1570C37200370DEA /* squirrel.yaml
+anchor=(88 87 squirrel.yaml)
+lastid=88
```
Comment (sih4sing5hong5) [inline package/add_data_files:16]: `id` kah `ref` 是欲tshòng sannh--ê？

File: ThoanTaigi.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ngoohebi) [reply ThoanTaigi.xcodeproj/project.pbxproj:274]: Screen Shot 2019-12-31 at 3 27 10 PM

Comment (ngoohebi) [reply ThoanTaigi.xcodeproj/project.pbxproj:274]: Che sī Xcode choan-àn koán-lí folder kap tóng-àn ê hong-sek.

File: package/add_data_files (hunk 2 above)
Comment (ngoohebi) [reply package/add_data_files:16]: To̍h-sī bô-kâng só͘-chai ê pian-bé niâ, che sī Xcode ka-tī sán-seng--ê.
Góa chiàu goân-té i kái ê hong-sek khì kái--ê.

File: ThoanTaigi.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ngoohebi) [reply ThoanTaigi.xcodeproj/project.pbxproj:274]: "plum" chín í-keng bô tī lāi-té, tô͘ sī hō͘ lí khòaⁿ--ê niâ.
"add_data_file" chit-ki thêng-sek ē kā ka--ji̍p-khì.
Cháu liáu khui Xcode --khí-lâi to̍h khòaⁿ ē tio̍h.

**Review by sih4sing5hong5 — APPROVED** [review]
看CI ê .pbxproj 是有tsinn--入-去，試過bô問題就merge

**Comment by ngoohebi** [comment]
Koh têng chhì 1-pái bô būn-tê.

**Pull request merged**